lib/mindwendel/permissions.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Mindwendel.Permissions do
+  @moduledoc """
+  The Permissions context.
+  """
+  def has_moderating_permission(brainstorming, current_user) when current_user != nil do
```

Comment:
I think I'd prefer "positive matches" - i.e. checking for what you think it is not what it isn't. F.ex. the String "null" would slip through here and cause an error.

Hence doing a pattern match on something like `%User{} = current_user` in the function head is more assertive about what you expect the data to be like